src/igvfd/schemas/prediction.json

```diff
@@ -48,6 +48,12 @@
         "schema_version": {
             "default": "3"
         },
+        "award": {
```

Comment:
add comment explaining that restriction is in place to drive towards deprecation ofthe schema

Reply:
Added.